src/styles.scss

```diff
@@ -1 +1,9 @@
 /* You can add global styles to this file, and also import other style files */
+
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap');
```

Comment:
best practices to download the `font` and put it into application `assets` folder.